pubsub/consts.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package pubsub
+
+import (
+	"time"
+
+	"github.com/ava-labs/avalanchego/utils/units"
+)
+
+const (
```

Comment:
You need to make these configurable when constructing the server. Many of these are insufficient.